WordPress/src/main/java/org/wordpress/android/ui/prefs/WPComSiteSettings.java

```diff
@@ -653,6 +653,10 @@ private void onSaveResponseReceived(Exception error) {
     private void deserializeWpComRestResponse(SiteModel site, JSONObject response) {
         if (site == null || response == null) return;
         JSONObject settingsObject = response.optJSONObject("settings");
+        if (settingsObject == null) {
+            AppLog.w(AppLog.T.API, "Error: Settings response doesn't contain settings object");
```

Comment:
Not a big deal, but I'm curious why you decided to go with a warning instead of an error. It seems like this is a pretty exceptional case that shouldn't really ever happen (unless you're the one user who seems to get this error all the time). This certainly is not a blocker.

Reply:
I was following the same approach as we have in the `onFailure` callback when requesting settings. However, I think you're right so I'm changing it 👍